src/app/analyze/stack/stack-details/stack-details.module.ts

```diff
@@ -1,14 +1,24 @@
-import { NgModule }         from '@angular/core';
-import { CommonModule }     from '@angular/common';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
 import { HttpModule, Http } from '@angular/http';
+import { DataTableModule } from 'angular2-datatable';
 
-import { StackDetailsComponent }     from './stack-details.component';
+import { StackDetailsComponent } from './stack-details.component';
+import { ModalModule } from 'ng2-modal';
```

Comment:
Please change 'ng2-model' to 'ngx-model' and run 'npm run reinstall'. The library was updated and they changed the name.